zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/signal/BroadcastSignalMultiplePartitionsTest.java

```diff
@@ -203,6 +203,17 @@ public void shouldRejectBroadcastForOneUnauthorizedProcess() {
                         .formatted(otherProcessId))
                 .exists())
         .isTrue();
+
+    // then - the command is still acknowledged because a redistribution is not necessary
+    assertThat(
+            RecordingExporter.commandDistributionRecords(CommandDistributionIntent.ACKNOWLEDGE)
```

Comment:
🔧 It would be more readable to like this imo:

```
    assertThat(
            RecordingExporter.commandDistributionRecords(CommandDistributionIntent.ACKNOWLEDGE)
                .withDistributionValueType(ValueType.SIGNAL)
                .withDistributionPartitionId(2)
                .withDistributionIntent(SignalIntent.BROADCAST)
                .exists())
        .isTrue();
```